README.md

```diff
@@ -1,138 +1,126 @@
+    * Ubuntu 18.04
+    * Debian 9
+
+Other operating systems should work, but they are not supported as the CyberPatriot / CyberCenturion competitions do not use them.
+
+## Installing
+Before CSSS will run, the following additional programs need to be installed onto your image:
+  * Windows:
+    * The .Net framework versions [4.0](https://www.microsoft.com/en-gb/download/details.aspx?id=17718) or [4.5](https://www.microsoft.com/en-gb/download/details.aspx?id=42642) (which one is required depends on if your image contains any of the needed Service Packs). Please follow the usual installation steps (Next, Next, ..., Finish)
+  * Linux:
+    * The Mono client should be installed. Please follow their instructions on the [Install Mono on Linux](http://www.mono-project.com/docs/getting-started/install/linux/) help article (summary: run the 3 lines of terminal commands from the top of the article, then run `sudo apt-get install -y mono-devel`)
+    * __For Debian images__, the libnotify package also needs to be installed (`sudo apt-get install libnotify-bin`)
+
+## Running
+Double-clicking on CSSS will cause it to quickly show the usage window, then close itself. This is by design. To run CSSS properly, a number of arguments need to be passed to the program, which are:
+```
+Usage:
+  CSSS.exe -c | -o | -p | -s | [-h] | [-m]
+
+Examples:
+  CSSS.exe -c
+  CSSS.exe -o -m
+  CSSS.exe -p
+  CSSS.exe -s
+  CSSS.exe -h
+
+
+Required arguments (at least one is needed):
+  -c, --check:    Checks the config files for any problems
+  -o, --observe:  Observes CSSS running before preparing it (implies '-c')
+  -p, --prepare:  Prepares CSSS ready for image release (implies '-c')
+  -s, --start:    Starts the scoring system
+
+
+Optional arguments:
+  -h, --help:     Shows this help message
+
+
+Developer arguments (all optional):
+  -m, --multiple: Allows multiple instances of CSSS to run concurently
+```
+
+The required arguments have been chosen to spell 'COPS', as competitors are 'policing' the security of the computer.
+
+Under normal image building, the `-o` option should be passed, as this allows CSSS to run as it would for training, but without affecting the files used to list the 'issues'.
+
+When you are ready to release the image, __with administrative privileges__ (e.g. `sudo`, "Run as administrator") run CSSS with the `-p` option to prepare the necessary files and allow CSSS to start automatically on computer reboots.
+> :warning: Running CSSS with the `-p` argument will encrypt the issue files. Make sure that you have a snapshot of your image before you run this command, otherwise you will need to set everything up again should there be any problems.
+
+## Contributing
+Thanks for your interest in CSSS. Read more about [contributing to the project](CONTRIBUTING.md) to get started.
+
+Please note that this project is released with a [Contributor Code of Conduct](#CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```

Comment:
As before, this is my fault. Could you remove the `#` from the link, so that it points to the file, rather than trying to find an anchor on the current page?